droidlet/memory/craftassist/mc_memory_nodes.py

```diff
@@ -165,13 +268,13 @@ def create(cls, memory, blocks: Sequence[Block]) -> str:
         # TODO this is going to cause a bug, need better way to initialize and track mean loc
         memory.db_write(cmd, memid, 0, 0, 0, "BlockObjects", 0)
         for block in blocks:
-            memory.upsert_block(block, memid, "BlockObjects")
-        memory.tag(memid, "_block_object")
-        memory.tag(memid, "_VOXEL_OBJECT")
-        memory.tag(memid, "_physical_object")
-        memory.tag(memid, "_destructible")
+            VoxelObjectNode.upsert_block(memory, block, memid, "BlockObjects")
+        memory.nodes["Triple"].tag(memory, memid, "_block_object")
+        memory.nodes["Triple"].tag(memory, memid, "VOXEL_OBJECT")
```

Comment:
a nit that this should be `_VOXEL_OBJECT`, we use `_`s to represent internal tags

Reply:
Thanks so much for catching this! This was the reason Voxel related tests were failing in my branch!